pages/docs/concepts/asyncapi-document/adding_messages.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Adding Messages
```

Comment:
🤔 "Adding messages" in the concepts bucket? Shouldn't it belong somewhere else? Or maybe we should change the title to something like "Messages"? @alequetzalli

Reply:
We already have message page - https://www.asyncapi.com/docs/concepts/message
This page is specifically written for **adding messages".

Reply:
But how is that a "concept"? Sorry if I'm not helping much. I don't know much about technical writing and trying to understand the context.

Reply:
Today, in our Spec 3 Docs call, I explained in detail to Fran the GSoD project scope. He should now understand why this is the `concepts` bucket and why we use diagrams in our project. If he forgets, he can review the recording 😄✌🏽

@fmvilas